config/admin_ranks.txt

```diff
@@ -30,14 +30,28 @@
 # +EVERYTHING (or +HOST or +ALL) = Simply gives you everything without having to type every flag
 # END_KEYWORDS
 
-Admin Observer
-Moderator		= +ADMIN
-Admin Candidate	= +@
-Trial Admin		= +@ +SPAWN +VAREDIT +BAN
-Badmin			= +@ +POSSESS +POLL +BUILDMODE +SERVER +FUN
-Game Admin		= +@ +STEALTH +SOUNDS +DEBUG
-Game Master		= +EVERYTHING
+## Trial Ranks
+Admin Observer 	    = +@
+Admin Candidate		= +@
 
-Host			= +EVERYTHING
 
-Coder			= +DEBUG +VAREDIT +SERVER +SPAWN +POLL
+## Junior Ranks
+TrialModerator		= +@ +BAN
+Moderator			= +@ +BAN
+
+
+## Admin Ranks
+Administrator		= +@ +STEALTH +DEBUG +SPAWN +VAREDIT +BAN
+Primarymin			= +@ +STEALTH +DEBUG +SPAWN +VAREDIT +POSSESS +POLL +FUN +BAN
+Seniormin			= +@ +STEALTH +SOUNDS +DEBUG +SPAWN +VAREDIT +POSSESS +POLL +BUILDMODE +FUN +BAN
+
+CommunityManager	= +EVERYTHING
```

Comment:
I feel like CM is a lesser rank to Headmin. Are you sure you want to give them +EVERYTHING?

Reply:
your CM should be either your host or your headmins